app/models/series.rb

```diff
@@ -104,4 +108,23 @@ def schedule_for_episode_number(episode_number)
 
     schedule_index
   end
+
+  def destroy
+    if v4?
+      really_destroy!
```

Comment:
This doesn't do any filtering (see the `paranoia_scope` at the top of the Story model).  This just causes any deletes of `v3` series to be paranoid, and deletes of `v4` series to be "real".